src/client/pages/create.js

```diff
@@ -188,13 +198,22 @@ class Create extends React.Component {
               </FormGroup>
             )}
             <hr />
+            <FormGroup row>
+              <Label for="gamemode" sm={2}>Gamemode</Label>
+              <Col sm={10}>
+                <Input id="gamemode" type="select" onChange={e => this.onGamemodeUpdated(e)} value={this.state.gamemode}>
+                  <option>EOP</option>
+                  <option>Cornucopia</option>
+                </Input>
+              </Col>
+            </FormGroup>
             <FormGroup row>
               <Label for="startSuit" sm={2}>Start Suit</Label>
               <Col sm={10}>
                 <Input type="select" name="startSuit" id="startSuit" onChange={e => this.onstartSuitUpdated(e)} value={this.state.startSuit}>
                   {
                     Object.keys(STARTING_CARD_MAP).map(suit => (
-                      <option value={suit} key={`start-suit-option-${suit}`}>{getTypeString(suit)}</option>
+                      <option value={suit} key={`start-suit-option-${suit}`}>{getTypeString((this.state.gamemode === "Cornucopia") ? "C" + suit : "" + suit)}</option>
```

Comment:
Maybe add the gameMode as a second parameter to getTypeString? I believe that might be better readable